pkg/provision/primitives/zdb.go

```diff
@@ -69,6 +69,10 @@ func (p *Provisioner) zdbProvisionImpl(ctx context.Context, reservation *provisi
 		return ZDBResult{}, errors.Wrap(err, "failed to decrypt namespace password")
 	}
 
+	if config.DiskType != pkg.HDDDevice {
```

Comment:
This is wrong. We still allow people to use 0-db on top of SSD if they want.
What we want to limit is the usage of HDD for anything else the 0-DB

Reply:
I understood that zdb can ONLY be HDD

Reply:
Okay, a bit misunderstanding ... we still need to support ssd and hdd for zdb

Reply:
So we drop this check?